backgroundfield/backgroundfield.cpp

```diff
@@ -30,10 +30,12 @@
 
 //FieldFunction should be initialized
 void setBackgroundField(
-   FieldFunction& bgFunction,
+   const FieldFunction& bgFunction,
    FsGrid< std::array<Real, fsgrids::bgbfield::N_BGB>, FS_STENCIL_WIDTH> & BgBGrid,
    bool append) {
    
+   using namespace std::placeholders;
+
```

Comment:
I would advocate not using that and putting that explicitly where it is needed. Is that for the _1, _2, _3? What are those?

Reply:
This is a "modern" (read: weird) mechanism in C++, in which you can take a function, pass it *some* arguments, leave other ones out, and obtain a function that only requires the remaining arguments.
Here, it is used to, for example, integrate over one dimension of a 3D function, returning a 2D function as a result.

So the alternative is to write ```std::placeholders::_1``` instead. That is probably less readable, but more understandable. :)